frontend/projects/upgrade/src/app/features/dashboard/feature-flags/modals/upsert-feature-flag-modal/upsert-feature-flag-modal.component.ts

```diff
@@ -184,7 +184,7 @@ export class UpsertFeatureFlagModalComponent {
       context: [appContext],
       tags,
       status: FEATURE_FLAG_STATUS.DISABLED,
-      filterMode: FILTER_MODE.INCLUDE_ALL,
+      filterMode: FILTER_MODE.EXCLUDE_ALL,
```

Comment:
we are setting exclude_all as default now?

Reply:
"exclude all" is right for the default. like experiments, we start with including nobody.

Reply:
Yes, I think that should be the default, just like how it works for experiments.